### PR TITLE
Changes after bug fix

### PR DESCRIPTION
**Pull request by ykuzakova**

### CODE REVIEW

File: plib/views/scripts/index/active-list.phtml [hunk 1]
```diff
@@ -5,20 +5,23 @@
 
     <h3><?php echo pm_Locale::lmsg('activeListHeader'); ?></h3>
 <?php echo $this->activeList([
-    'columnsNumber' => 1,
+    'layout' => [
+        'type' => pm_View_Helper_ActiveList::LAYOUT_RESPONSIVECOLUMN,
+        'stretched' => true,
+        'columns' =>  'xl-2 xxl-3',
+                ],
```
Comment (mdsina) [inline plib/views/scripts/index/active-list.phtml]: Align this brace with base padding near start of 'locale'

File: plib/views/scripts/index/active-list.phtml [hunk 2]
```diff
@@ -100,5 +108,28 @@
...
+            'toolbar' => [
+                [
+                    'title' => pm_Locale::lmsg('action4Title'),
+                    'iconClass' => 'icon-start',
+                ],
+            ],
+        ],
     ],
-]);
+]);
```
Comment (mdsina) [inline plib/views/scripts/index/active-list.phtml]: Add new line after end of the file

File: plib/resources/locales/en-US.php [hunk 3]
```diff
@@ -22,4 +26,4 @@
     'activeListHeader' => 'Your Containers',
     'htmlBlock' => 'Some html block for data',
     'toolbarTitle' => 'toolbar in service',
-];
+   ];
```
Comment (mdsina) [inline plib/resources/locales/en-US.php]: Unnecessary change

File: plib/views/scripts/index/active-list.phtml (hunk 1 above)
Comment (sibprogrammer) [inline plib/views/scripts/index/active-list.phtml]: should be one space after =>

File: plib/views/scripts/index/active-list.phtml [hunk 4]
```diff
@@ -100,5 +108,28 @@
                 ],
             ],
         ],
+        [
+            'id' => 'container-3',
+            //'icon' =>  pm_Context::getBaseUrl() . 'images/details.png',
```
Comment (sibprogrammer) [inline plib/views/scripts/index/active-list.phtml]: why is it commented out?
